src/Components/WarningComponent/WarningMessage.tsx

```diff
@@ -4,15 +4,26 @@ import ResultsTranslate from '../Results/Translate/Translate';
 import { Translation } from '../../Types/Results';
 
 type WarningMessageProps = {
-  message: Translation;
+  warning: {
+    message: Translation;
+    link_url: Translation;
+    link_text: Translation;
+  };
```

Comment:
Make this the warning message type from `Results.ts`
```suggestion
  warning: WarningMessage;
```